### ISSUE TITLE
Fails to disassemble FWAIT

### ISSUE THREAD

**Issue by carlreinke** [NationalSecurityAgency/ghidra#1003]
**Describe the bug**
In some contexts, the FWAIT instruction cannot be disassembled.

**To Reproduce**
1. Import the following in x86-16 Real Mode:
    ```
    0000:0000 9b d9 3e 0a       FSTCW        word ptr [0xa]
              00
    0000:0005 9b                WAIT
    0000:0006 8b 0e 0a 00       MOV          CX,word ptr [0xa]
    0000:000a 80 e5 f3          AND          CH,0xf3
    0000:000d 0b c1             OR           AX,CX
    0000:000f a3 1a 00          MOV          [0x1a],AX
    0000:0012 9b                WAIT
    0000:0013 d9 2e 1a 00       FLDCW        word ptr [0x1a]
    0000:0017 9b                ??           9Bh
    0000:0018 d9 fc             FRNDINT
    0000:001a 9b                WAIT
    0000:001b d9 2e 0a 00       FLDCW        word ptr [0xa]
    0000:001f c3                RET
    ```
2. Try to disassemble the `9B` at `0000:0017`.
3. Notice bookmark:

    | Type | Category | Description | Location | Label | Code Unit |
    | -- | -- | -- | -- | -- | -- |
    | Error | Bad Instruction | Unable to resolve constructor at 0000:0017 (flow from 0000:0013) | 0000:0017 |   | ?? 9Bh

**Expected behavior**
FWAIT instruction is disassembled.

**Environment:**
 - OS: Windows 10
 - Java Version: 12.0.2
 - Ghidra Version: 9.0.4

**Comment by mumbel** [comment]
Did you check master?  Can confirm for 9.0.4

It's narrowed down to `WAIT` and `FSTCW`, checks the pattern for fstcw first and matches the first two tokens and fails trying to resolve `m16`.  Seems like it should then do a `check pattern [2 of 2]:...` and go back to `WAIT`, but errors out instead.

**Comment by GhidorahRex** [comment]
This has been resolved by 2ab04ae86d2d4fc4e88c0055ee772bfabce0b394
